src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
```

Comment:
The `checkHealth` method is used to remove dead animals from the `Animal.alive` array. Consider renaming this method to more accurately reflect its purpose, such as `removeIfDead` or `updateAliveStatus`. This will improve code readability and maintainability.